city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,137 @@
+    name = "chi_ssa_27"
+    agency = "Chicago Special Service Area #27 Lakeview West"
+    timezone = "America/Chicago"
+    allowed_domains = ["lakeviewchamber.com"]
+    start_urls = ["https://www.lakeviewchamber.com/ssa27"]
+    minutes_list = []
+
+    def parse(self, response):
+        """   `parse` should always `yield` Meeting items. """
+        self.minutes_list = self.get_minutes_panel_items(response)
+        self._validate_locations(response)
+        commission_path = "div #content-232764 div.panel-body p"
+        for item in response.css(commission_path)[1:]:  # main
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description='',
+                classification='',
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes='',
+                location=self._parse_location(),
+                links=self._parse_links(item),
+                source=response.url,
+            )
+            meeting['status'] = self._get_status(meeting)
+            meeting['id'] = self._get_id(meeting)
+            yield meeting
+
+    def clean_up_title(self, t_name=''):
+        if len(t_name) > 0:
+            while t_name[0:1] in ['\n', '(']:
+                t_name = t_name[1:]
+            if t_name[-1] == ')':
+                t_name = t_name[:-1]
+        return t_name
+
+    def get_minutes_panel_items(self, response):
+        class Paragraph(Item):
+            link = Field()
+            date = Field()
+            date2 = Field()
+            title = Field()
+
+        panel = response.css("#content-232768 div.panel-body p")[1:]
+        min_list = []
+        paragraphs = panel.css('p')
+
+        last_paragraph = paragraphs[-1].css('*::text').getall()
+        if '\xa0' in last_paragraph:
+            paragraphs = paragraphs[:-1]
+
+        first_paragraph = paragraphs[0:1].css('*::text').getall()
+        if '\xa0' in first_paragraph:
+            paragraphs = paragraphs[1:]
+
+        for p in paragraphs:
+            t_name = ''
+            href = p.css('a::attr(href)').get()
+            t_list = p.css('*::text').getall()
+            dt_date = datetime.strptime(t_list[0], '%B %d, %Y')
+            try:
+                t_name = t_list[1]
+                if t_name == '':
+                    t_name = (split('-', split('/', href)[-1], 3)[-1])[:-4]
+            except IndexError:
+                pass
+
+            t_name = self.clean_up_title(t_name)
+            dt_two = dt_date.date()
+            min_list.append(Paragraph(link=href, date=dt_date, date2=dt_two, title=t_name))
+        return min_list
+
+    def _parse_links(self, item):
+        item_txt = ' '.join(item.css('*::text').extract()).strip()
+        item_txt = sub("Annual Meeting", "", item_txt)
+        item_txt = item_txt.replace("June", 'Jun').replace("Sept", 'Sep')
+
+        loc_items = split(',', item_txt, 4)
```

Comment:
If you're just splitting on a comma, you won't need the regex function and can just do `item_text.split(',')`

Reply:
Thank you, - done.